offload/plugins-nextgen/host/src/rtl.cpp

```diff
@@ -440,6 +440,8 @@ struct GenELF64PluginTy final : public GenericPluginTy {
 #else
     return llvm::Triple::ppc64;
 #endif
+#elif defined(__riscv) && (__riscv_xlen == 64)
+    return llvm::Triple::ArchType::riscv64;
```

Comment:
Is `ArchType` necessary here?

Reply:
Indeed good catch, I don't think so. I have launched a test build to confirm.

Reply:
I confirm this works fine, so I have just pushed an updated version.